src/globus_sdk/tokenstorage/base.py

```diff
@@ -23,12 +26,19 @@ def get_token_data(self, resource_server: str) -> dict[str, t.Any] | None:
         Either returns a dict with the access token, refresh token (optional), and
         expiration time, or returns ``None``, indicating that there was no data for that
         resource server.
+
+        :param resource_server: The resource_server string which uniquely identifies the
+            token data to retriever from storage
+        :type resource_server: str
         """
 
     def on_refresh(self, token_response: OAuthTokenResponse) -> None:
         """
         By default, the on_refresh handler for a token storage adapter simply
         stores the token response.
+
+        :param token_response: The token response received from the refresh
+        :type token_response: :class:`OAuthTokenResponse`
```

Comment:
This one [doesn't render correctly](https://globus-sdk-python--835.org.readthedocs.build/en/835/tokenstorage.html#globus_sdk.tokenstorage.StorageAdapter.on_refresh).

```suggestion
        :type token_response: :class:`~.OAuthTokenResponse`
```